cmd/algons/dnsCmd.go

```diff
@@ -293,26 +318,48 @@ func doDeleteDNS(network string, noPrompt bool, excludePattern string) bool {
 	cloudflareDNS := cloudflare.NewDNS(cfZoneID, cfEmail, cfKey)
 
 	idsToDelete := make(map[string]string) // Maps record ID to Name
+	services := []string{"_algobootstrap", "_metrics"}
+	servicesRegexp, err := regexp.Compile("(algobootstrap|_metrics)._tcp.*algodev.network")
```

Comment:
that can't be correct... I think that you meant
`(_algobootstrap|_metrics)` and you want to have a <network name> between the tcp and the algodev.

Reply:
I guess this appears to be working because `MatchString` allows substring matches by default. I'll fix the missing underscore, and wrap the thing in `^...$`